example/efp_gpt/parallel_main_efp.cpp

```diff
@@ -30,7 +30,7 @@ int main(int argc, char *argv[]) {
         return -1;
     }
     auto model = std::make_shared<EFP>("efp", jobPeriod, processingTime, obsTime);
-    model->flatten();
+    // model->flatten();
```

Comment:
Is this necessary or does the parallel coordinator flatten the model automatically?